backend/TaskTree/Models/Requests/UpdateTaskRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using TaskTree.Models.Responses;
+
+namespace TaskTree.Models.Requests
+{
+    public class UpdateTaskRequest
+    {
+        public string? Name { get; set; } = null!;
```

Comment:
same as above, when updating a project through the api, there will probably only be updates to name, description, weight and completed at.